.github/workflows/react_demos.yml

```diff
@@ -0,0 +1,56 @@
+name: React Demos
+
+on:
+  push:
```

Comment:
This lets us manually trigger the actions from GitHub and is a huge help when testing new actions
```suggestion
  workflow_dispatch:
  push:
```

Reply:
Good idea, i would create a dummy pull request for such a thing!